examples/classification/script.jl

```diff
@@ -0,0 +1,184 @@
+# # Stochastic Variational Classification
+#
+# This example demonstrates how to carry out non-conjugate Gaussian process
+# inference using the stochastic variational Gaussian process (SVGP) model. For
+# a basic introduction to the functionality of this library, please refer to the
+# [User Guide](@ref).
+#
+# ## Setup
+
+using SparseGPs
+using ParameterHandling
+using Zygote
+using Distributions
+using LinearAlgebra
+using Optim
+
+using Plots
+default(; legend=:outertopright, size=(700, 400))
+
+using Random
+Random.seed!(1234)
+#md nothing #hide
+
+# ## Generate some training data
+#
+# For our binary classification model, we will use the standard approach of a
+# latent GP with a Bernoulli likelihood. This results in a generative model that
+# we can use to produce some training data.
+#
+# First, we define the underlying latent GP
+# ```math
+# f \sim \mathcal{GP}(0, k(\cdot, \cdot'))
+# ```
+# and sample a function `f`.
+
+k_true = [30.0, 0.5]
+kernel_true = k_true[1] * (SqExponentialKernel() ∘ ScaleTransform(k_true[2]))
+
+lgp = LatentGP(GP(kernel_true), BernoulliLikelihood(), 1e-12)
```

Comment:
would be helpful to explain the 1e-12 here again
```suggestion
jitter = 1e-12  # for numeric stability
lgp = LatentGP(GP(kernel_true), BernoulliLikelihood(), jitter)
```
along those lines..:)